specs/MultiProfile.md

```diff
@@ -169,20 +169,37 @@ void ScenarioCookieManagement::DeleteAllCookies()
 ### Delete profile
 
 ```cpp
-HRESULT AppWindow::DeleteProfile(ICoreWebView2Controller* controller)
+HRESULT AppWindow::DeleteProfile(ICoreWebView2* webView2)
 {
-    wil::com_ptr<ICoreWebView2> coreWebView2;
-    CHECK_FAILURE(controller->get_CoreWebView2(&coreWebView2));
-    auto webview7 = coreWebView2.try_query<ICoreWebView2_7>();
-    if (webview7)
+    DCHECK(webView2);
```

Comment:
Just curious, what are best practices for using debug checks in sample code?  It seems OK but I don't think I see this often.

Reply:
will remove `DCHECK(webView2);` and change to `auto webview13 = wil::try_com_query<ICoreWebView2_13>(webview2);`